src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__text {
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $textColorBlack;
+  }
+
+  &__code {
+    margin-top: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
font-weight: 400; is a default browser value. No needs to set it again. remove from similar cases as wel
```suggestion
```